app/services/rights_labeler.rb

```diff
@@ -2,48 +2,35 @@
 
 # Parses rights XML to produce a label
 class RightsLabeler
-  # @param [String] rights_xml rights metadata serialized as XML in string form
+  # @param [Cocina::Models::AdminPolicyDefaultAccess] default_access admin policy default access instance
   # @return [String] a label expressing the rights of the object
-  def self.label(rights_xml)
-    new(rights_xml).label
+  def self.label(default_access)
+    new(default_access).label
   end
 
-  # @param [String] rights_xml rights metadata serialized as XML in string form
-  def initialize(rights_xml)
-    @rights_xml = Nokogiri::XML(rights_xml)
+  # @param [Cocina::Models::AdminPolicyDefaultAccess] default_access admin policy default access instance
+  def initialize(default_access)
+    @default_access = default_access
   end
 
   # @return [String] a label expressing the rights of the object
   def label
-    machine_read_access_node = rights_xml.search('//rightsMetadata/access[@type="read"]/machine').first
-    machine_discover_access_node = rights_xml.search('//rightsMetadata/access[@type="discover"]/machine').first
+    return '' if default_access.nil? || default_access.access.nil?
 
-    if machine_read_access_node && machine_read_access_node.search('./group[text()="Stanford" or text()="stanford"]').size.positive?
-      if machine_read_access_node.search('./group[@rule="no-download"]').size.positive?
-        'stanford-nd'
-      else
-        'stanford'
-      end
-    elsif machine_read_access_node && machine_read_access_node.search('./world').size.positive?
-      if machine_read_access_node.search('./world[@rule="no-download"]').size.positive?
-        'world-nd'
-      else
-        'world'
-      end
-    elsif machine_read_access_node && machine_read_access_node.search('./location[text()="spec"]').size.positive?
-      'loc:spec'
-    elsif machine_read_access_node && machine_read_access_node.search('./location[text()="music"]').size.positive?
-      'loc:music'
-    elsif machine_discover_access_node && machine_discover_access_node.search('./world').size.positive?
-      # if it's not stanford restricted, world readable, or location restricted, but it is world discoverable, it's "citation only"
-      'citation-only'
-    elsif machine_discover_access_node && machine_discover_access_node.search('./none').size.positive?
-      # if it's not even discoverable, it's "dark"
-      'dark'
+    if default_access&.controlledDigitalLending
+      'cdl-stanford-nd'
```

Comment:
This is NEW CODE that I added because I thought we were missing a valid case, but if this is not needed, I am happy to remove it.

Reply:
Calling this out for @justinlittman @jcoyne @andrewjbtw